topdown/crypto.go

```diff
@@ -205,6 +209,55 @@ func builtinCryptoX509ParseRSAPrivateKey(_ BuiltinContext, operands []*ast.Term,
 	return iter(ast.NewTerm(value))
 }
 
+func builtinCryptoParsePrivateKey(_ BuiltinContext, operands []*ast.Term, iter func(*ast.Term) error) error {
+
+	a := operands[0].Value
+	input, err := builtins.StringOperand(a, 1)
+	if err != nil {
+		return err
+	}
+
+	// get the raw private key
+	rawKey, err := getRSAPrivateKeyFromString(string(input))
+	if err != nil {
+		return err
+	}
+	key, err := parsePrivateKey(rawKey)
+
+	theJsonKey, err := json.Marshal(key)
```

Comment:
suggestion: elsewhere in the project we generally use the convention `bs` for the bytes created as the result of JSON marshalling.

```suggestion
	bs, err := json.Marshal(key)
```